armi/reactor/blockParameters.py

```diff
@@ -93,14 +93,12 @@ def detailedNDens(self, value):
             "molesHmBOL",
             units="mole",
             description="Total number of atoms of heavy metal at BOL assuming a full assembly",
-            default=None,
         )
 
         pb.defParam(
             "massHmBOL",
             units="grams",
             description="Mass of heavy metal at BOL",
-            default=None,
```

Comment:
These are scalar values, so the default should be 0.0 rather than `None`. I don't have it documented, but I encountered errors with the defaults as `None`.